[DOCS] More updates to the how to get support page (#9818)
great-expectations/great_expectations: docs/docusaurus/versioned_docs/version-0.18/resources/get_support.md

description: Learn what resources are available to help you resolve issues with GX Cloud and GX OSS.
---

> Using the appropriate support channels can save time and improve response times.

Reviewer: ```suggestion
Knowing how and where to request GX support can reduce downtime and improve response times.
```

## How to effectively get support

> Strategies to ensure your questions get resolved quickly.

Reviewer: ```suggestion
The following are some recommendations for resolving issues on your own. Resolving issues on your own builds your knowledge and allows you to share that knowledge with the community.
```

> ### Try solving your issue first

Reviewer: ```suggestion

```

Reviewer: Remove the text (_The following are..._) as well.
